model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -238,7 +238,12 @@ class DependencyGraphBuilder<D>(
      * Update the dependency graph by adding the given [dependency], which may be [transitive], for the scope with name
      * [scopeName]. All the dependencies of this dependency are processed recursively.
      */
-    private fun addDependencyToGraph(scopeName: String, dependency: D, transitive: Boolean): DependencyReference {
+    private fun addDependencyToGraph(
+        scopeName: String,
+        dependency: D,
+        transitive: Boolean,
+        processed: Set<D>
```

Comment:
Please mention this new parameter in the function docs.

Reply:
Or probably it's enough to document it only for the most low-level function for which this new parameter was introduced.

Reply:
The function docs have been extended.